docker/Dockerfile

```diff
@@ -50,4 +50,4 @@ COPY --from=rake /usr/lib/x86_64-linux-gnu/libicu* /usr/lib/x86_64-linux-gnu/
 WORKDIR /srv/app
 
 EXPOSE 1791
-CMD ["bundle", "exec", "rake", "server"]
+CMD ["bundle", "exec", "rackup", "-p", "1791"]
```

Comment:
some diff/changes are missing from this commit : https://github.com/Mapotempo/geocoder-api/commit/c8bb85427706723ff1d1e7181f8481b3250524d8